koku/sources/management/commands/sources_listener.py

```diff
@@ -0,0 +1,42 @@
+#
```

Comment:
This file enables us to separate the sources-api and the sources-listener into separate pods. This separation will likely happen with Clowder, not e2e.